### PR TITLE
Feature/day planner

### PR DESCRIPTION
**Pull request by baywet**
Hey @BrianTJackett @gavinbarron 
Here is a first draft for the Planner stuff.
My biggest road block at the moment: planner APIs don't support application permissions so I'd have to implement a device code flow or something similar.
But I wanted to get your feedback before diving into that.

Also currently targeting the reorganize branch, let me know if I should change the target. (started of the feature/day-template)

### CODE REVIEW

**Comment by baywet** [comment]
TODOs:
- update screenshots for delegated permissions
- include directory.read.all permission
- document version bump for MS Identity
- document new provider addition
- update code samples
- document creating a group as a placeholder in outlook

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Each of the samples builds from a common base project, in order to sucessfully b
 ### Console Applications
 - [Base console application setup](./base-console-app)
 - [Day 16 - Create a new user](./day16-create-user)
+- [Day NN - Template](./dayNN-template)
```
Comment (gavinbarron) [inline README.md]: Can you please update this label in the TOC to use the correct day number and scenario

**Comment by gavinbarron** [comment]
Thanks @baywet, can you please rename the dayNN-template folder to match the day that this falls on?

File: README.md (hunk 1 above)
Comment (baywet) [reply README.md]: We had an email thread going with @BrianTJackett I still don't know what my day number is. Once I get the information I'll update it.

Comment (BrianTJackett) [reply README.md]: @baywet In the current schedule Planner is on Day 22, but don't worry too much about the exact day number.  We have been shifting days around based on readiness of content.  If yours is ready to go sooner we may move it up in the schedule.  If you want to stick with a generic "Day NN - Planner" for time being I'm fine with that.

**Pull request merged**